app/models/solidus_paypal_commerce_platform/payment_source.rb

```diff
@@ -3,11 +3,11 @@
 module SolidusPaypalCommercePlatform
   class PaymentSource < ::Spree::PaymentSource
     self.table_name = "paypal_commerce_platform_sources"
-    enum paypal_funding_source: {
+    enum paypal_funding_source, {
```

Comment:
I think we still need to pass in a symbol for the enum.
```suggestion
    enum :paypal_funding_source, {
```

Reply:
Dang. Thanks